backend/src/ml_space_lambda/emr/lambda_functions.py

```diff
@@ -225,6 +225,47 @@ def list_all(event, context):
     return query_resource_metadata(resource_metadata_dao, event, ResourceType.EMR_CLUSTER)
 
 
+@api_wrapper
+def list_applications(event, context):
+    # EMR doesn't seem to have an API for listing the applications available for a specific release label.
+    # If you describe a release label you can get the applications it supports, but we just want to display
```

Comment:
When creating an EMR cluster you choose a release label (ex 6.2.0, 6.10.0, etc)

You can describe a release label to get the list of applications for *that* release label, but there's no way to list all applications (since they vary slightly based on release label).

This is a list of applications that are supported for all 6.X releases (the only ones we're supporting in MLSpace).